README.md

```
@@ -95,6 +95,41 @@ menv make brew-formulae mbk -o  # Force overwrite configs
+| `ollama` | `ol` | LLM |
+| `ollama-models` | `ol-m` | LLM |
+
+Examples:
+
+```sh
+menv make agy
+menv make nd-p
+menv make brew-f mbk
+menv make ol-m
+```
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)